### PR TITLE
#60 Frontend changes using gulp

### PR DESCRIPTION
**Pull request by pragatikanade**
Implemented  SCSS Folder structure using gulp.
created gulpfile.js  file which contains function to convert scss to css.

### CODE REVIEW

**Comment by pragatikanade** [comment]
@rahulsavaria please review PR.

**Review by guptahemant — APPROVED** [review]

File: public-html/searchbook.php [hunk 1]
```diff
@@ -30,7 +30,13 @@
     $menu->render_header();
     $menu->render_menu();
     ?>
-    <div class="container search-div">
+    <div class="container">
+		<div class="main">
+        <div class="header">
+			<h3>Filter by:</h3>
```
Comment (shriaas2898) [inline public-html/searchbook.php:36]: @pragzii3896  I can see 2 `Filter By` after submitting the form:
image
Can you please fix this?

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+//const gulp = require("gulp")
+//const sass = require("gulp-sass")
+//const watchSass = require("gulp-watch-sass") 
+//gulp.task("sass:watch", () => watchSass([
+  //"./public-html/asset/**/*.{scss,css}",
+  //"!./public/libs/**/*"
+//])
+  //.pipe(sass())
+  //.pipe(gulp.dest("./public")));
```
Comment (shriaas2898) [inline gulpfile.js:33]: Hey if this code is not being used anywhere can you remove it?

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+//function defaultTask(cb) {
+ // place code for your default task here
+ //cb()
+//}
+//exports.default = defaultTask
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+gulp.task('hello', function() {
```
Comment (shriaas2898) [inline gulpfile.js:8]: I guess this task is for testing, do we really need it on the master branch?

**Review by shriaas2898 — COMMENTED** [review]
Hey most of the things look great! There are somethings which can be removed/ modified please check that. Also it would be awesome if you could add steps to use `gulp` in the `README.md` so that everyone can follow that.

File: gulpfile.js (hunk 2 above)
Comment (shriaas2898) [reply gulpfile.js:33]: And since this file is a `.js` file it should be inside `js` folder right?

**Pull request merged**